packages/design-system/src/components/web-components/ds-autocomplete/ds-autocomplete.test.tsx

```diff
@@ -1,8 +1,10 @@
-import { render, waitFor, screen } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import { config } from '../../config';
 import userEvent from '@testing-library/user-event';
 import './ds-autocomplete';
 
+jest.mock('lodash/uniqueId', () => jest.fn((prefix) => `${prefix || 'id'}-test`));
```

Comment:
Can this be achieved by passing a `root-id` to the component? Having a static id for the snapshots?